fireblocks_sdk/sdk.py

```diff
@@ -51,16 +84,177 @@ def __init__(self, private_key, api_key, api_base_url="https://api.fireblocks.io
         self.token_provider = SdkTokenProvider(private_key, api_key, seconds_jwt_exp)
         self.timeout = timeout
         self.http_session = requests.Session()
-        self.http_session.headers.update({
-            'X-API-Key': self.api_key,
-            'User-Agent': self._get_user_agent(anonymous_platform)
-        })
+        self.http_session.headers.update(
+            {
+                "X-API-Key": self.api_key,
+                "User-Agent": self._get_user_agent(anonymous_platform),
+            }
+        )
+        self.NCW = self._create_ncw()
+
+    def _create_ncw(self):
+        return self.NCW(self)
+
+    class NCW:
```

Comment:
I think we can extract this class to a different file. it gets the SDK as arg, so no real reason to define it under the SDK class. To initiate it you'll still have to initiate the SDK first and pass it in the constructor.